### DISCUSSION TITLE
github

### DISCUSSION THREAD

**Discussion by nandini965** [learndevopsonline/learndevopsonline#1673] [Q&A]
hi sir @r-devops , my push access got denied
![image](https://github.com/learndevopsonline/learndevopsonline/assets/129321520/51b4a00e-5c4d-4d6a-add4-f7f315cda967)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/129321520/0c18a608-8023-4531-9ee1-fd15d0f8a4b7)

**Comment by sreedharm07** [comment]
@nandini965 
just click on authorize in github which you see in screen 
or generate tocken in github and login with token to intillij

**Reply by nandini965** [reply]
okay

**Reply by nandini965** [reply]
when i clicked on authorize , i got this mail 
![image](https://github.com/learndevopsonline/learndevopsonline/assets/129321520/277c414a-d611-4150-83dd-75e3fb696fa9)
and, i didnot see any token to generate ,it automatically gave access to the push

**Reply by sreedharm07** [reply]
yes thats it its taken by login 
you authenticated with github, creating token is another method.
now you can push code from intilij 

if you are not getting the point still please join lab session, lab session is now running whole day, you can join even now and you can clarify your issue

**Reply by nandini965** [reply]
ohh okay,  i got u and thanq for the information
